traffic_ops/traffic_ops_golang/types/types_test.go

```diff
@@ -168,18 +168,6 @@ func TestCreateInvalidType(t *testing.T) {
 	}
 }
 
-func TestDeleteInvalidType(t *testing.T) {
-	invalidDeleteType := createDummyType("other")
-
-	err, _, statusCode := invalidDeleteType.Delete()
-	if err == nil {
-		t.Fatalf("expected delete type to have an error")
-	}
-	if statusCode != http.StatusNotFound {
```

Comment:
No need to take this test out, the comparison should just be against `http.StatusBadRequest`

Reply:
Originally I thought the test no longer was testing anything other than the generic api method, but you are correct; I didn't need to remove it.